src/Xamarin.Android.Build.Tasks/Linker/MonoDroid.Tuner/Extensions.cs

```diff
@@ -11,6 +11,10 @@
 using Java.Interop;
 using Java.Interop.Tools.Cecil;
 
+#if NET5_LINKER
+using Microsoft.Android.Sdk.ILLink;
+#endif
```

Comment:
(Attempted) convention: please use a comment on `#else`/`#endif` with the same expression as the `#if` they correspond to, e.g.

```csharp
#if NET5_LINKER
…
#else   // !NET5_LINKER
…
#endif  // !NET5_LINKER
```